dev-games/ogre/ogre-1.10.12.ebuild

```diff
@@ -13,41 +15,46 @@ LICENSE="MIT public-domain"
 SLOT="0/1.10.0"
 KEYWORDS="~amd64 ~arm ~x86"
 
-IUSE="cg doc double-precision examples +freeimage gl3plus gles2 gles3 ois +opengl profile tools"
+IUSE="beta-components +cache cg doc double-precision egl examples +freeimage
+	gles2 ois +opengl profile +resman-legacy resman-pedantic resman-strict
+	tools"
 
-REQUIRED_USE="examples? ( ois )
-	gles3? ( gles2 )
-	gl3plus? ( opengl )"
+REQUIRED_USE="
+	|| ( gles2 opengl )
+	^^ ( resman-legacy resman-pedantic resman-strict )
+	examples? (
+		beta-components
+		ois
+	)"
 
 RESTRICT="test" #139905
 
 RDEPEND="
 	dev-libs/boost
```

Comment:
That is missing a `:=` subslot operator.

Reply:
fixed